tidb-cloud/scale-tidb-cluster.md

```diff
@@ -11,22 +11,67 @@ aliases: ['/tidbcloud/beta/scale-tidb-cluter']
 > - Currently, you cannot scale a [Developer Tier cluster](/tidb-cloud/select-cluster-tier.md#developer-tier).
 > - When a cluster is in the scaling status, you cannot perform any new scaling operations on it.
 
-## Scale a cluster
+You can scale a TiDB cluster in the following dimensions:
 
-To scale a cluster, take the following steps:
+- Node number of TiDB, TiKV, and TiFlash
+- Storage size of TiKV and TiFlash
+- Node size (including vCPUs and memory) of TiDB, TiKV, and TiFlash
 
-1. Navigate to **Active Clusters** page and click the name of a cluster that you want to scale. The overview page of the cluster is displayed.
+For information about how to determine the size of your TiDB cluster, see [Determine Your TiDB Size](/tidb-cloud/size-your-cluster.md).
+
+## Change node number
+
+### Increase node number
+
+To increase the number of TiDB, TiKV, or TiFlash nodes, take the following steps:
+
+1. In the TiDB Cloud console, navigate to the **Active Clusters** page for your project, and then click the name of a cluster that you want to scale. The overview page of the cluster is displayed.
 2. In the cluster information pane on the left, click **Setting**.
 3. Click **Scale** in the drop-down menu. The **Scale** window is displayed.
-4. In the **Scale** window, you can make the following changes:
+4. In the **Scale** window, increase the number of TiDB, TiKV, or TiFlash nodes.
+5. Click **Confirm**.
+
+### Decrease node number
 
-    - TiDB: increase or decrease the node quantity.
-    - TiKV: increase the node quantity and storage size.
-    - TiFlash: increase the node quantity and storage size.
+To decrease the number of TiDB nodes, take the following steps:
 
+1. In the TiDB Cloud console, navigate to the **Active Clusters** page for your project, and then click the name of a cluster that you want to scale. The overview page of the cluster is displayed.
+2. In the cluster information pane on the left, click **Setting**.
+3. Click **Scale** in the drop-down menu. The **Scale** window is displayed.
+4. In the **Scale** window, decrease the number of TiDB nodes.
 5. Click **Confirm**.
 
-## Increase the node size
+To decrease the number of TiKV or TiFlash nodes, you need to submit a support ticket. The PingCAP support team will contact you and complete the scaling within the agreed time.
+
+> **Warning:**
+>
+> Decreasing TiKV or TiFlash node number can be risky, which might lead to insufficient storage space, excessive CPU usage, or excessive memory usage on remaining nodes.
+
+To submit a support ticket, perform the steps in [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md). For each node to be scaled, provide the following information in the **Description** box:
+
+- Cluster name: xxx
+- Cloud provider: GCP or AWS
+- Node type: TiKV or TiFlash
+- Current node number: xxx
+- Expected node number: xxx
+
+## Change storage size
+
```

Comment:
Please add a short description between the two headings.

Reply:
Thanks for the comment. The short description is added now.

Reply:
LGTM. Thanks~